prow/config/org/org.go

```diff
@@ -22,6 +22,12 @@ import (
 	"k8s.io/test-infra/prow/github"
 )
 
+// FullConfig stores the full configuration to be used by the tool, mapping
+// orgs to their configuration at the top level under an `orgs` key.
+type FullConfig struct {
+	Orgs map[string]*Config `json:"orgs,omitempty"`
```

Comment:
Why did you switch this to a pointer? I think it should remain as `map[string]Config`.

Reply:
Oops, I copied the full config export logic, which was also a pointer. Weird.